x-pack/plugin/deprecation/qa/early-deprecation-rest/build.gradle

```diff
@@ -27,9 +27,12 @@ restResources {
 
 testClusters.configureEach {
   testDistribution = 'DEFAULT'
-  setting 'cluster.deprecation_indexing.enabled', 'true'
   setting 'xpack.security.enabled', 'false'
   setting 'xpack.license.self_generated.type', 'trial'
+  setting 'cluster.deprecation_indexing.enabled', 'true'
+  setting 'cluster.deprecation_indexing.flush_interval', '1ms'
```

Comment:
This seems infeasibly short. Would 100ms be a bit closer to real-world usage?

Reply:
the smallest the interval the better. We want the flush happen as soon as possible (to make sure it will be before the templates are loaded) 
On my laptop it takes <500ms from the log event being submitted to bulk processor to loading the templates. Could be that on CI machine it would be faster?

Reply:
> disrupt normal ES working

this is the goal. indexing should not happen that early in startup so there is a queue of those too early requests to process later